### DISCUSSION TITLE
How to include JSON files in build step

### DISCUSSION THREAD

**Discussion by mahdiboomeri** [nuxt/nuxt#16774] [Questions]
### Context
nuxt version:  3.0.0-rc.8

### Description
I have a server route for receiving information directly from a set of JSON files in my filesystem. Currently i have a single route which takes a query arg and base on that I return the correct file using node's `fs.readFile()`. While this approach works in development after the build step my JSON files don't get included in the final bundle.
How do i configure Vite in `nuxt.config.ts` to make sure that those files are included in my server bundle.

### Possible solutions
Creating a seperate route for each file instead of relying on one singular route: This is not suitable for my usecase as i have over 5,000 files that i have to serve.

**Comment by sevillaarvin** [comment]
Does `import myData from "./data.json"` work?
